streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableAggregateTest.java

```diff
@@ -162,83 +175,105 @@ public void testAggRepartition() {
                             return KeyValue.pair(value, value);
                     }
                 },
-                stringSerialzied)
+                stringSerialized)
             .aggregate(
                 MockInitializer.STRING_INIT,
                 MockAggregator.TOSTRING_ADDER,
                 MockAggregator.TOSTRING_REMOVER,
                 Materialized.<String, String, KeyValueStore<Bytes, byte[]>>as("topic1-Canonized")
                     .withValueSerde(stringSerde));
 
+        final MockProcessorSupplier<String, Object> supplier = new MockProcessorSupplier<>();
         table2.toStream().process(supplier);
 
-        driver.setUp(builder, stateDir);
-
-        driver.process(topic1, "A", "1");
-        driver.flushState();
-        driver.process(topic1, "A", null);
-        driver.flushState();
-        driver.process(topic1, "A", "1");
-        driver.flushState();
-        driver.process(topic1, "B", "2");
-        driver.flushState();
-        driver.process(topic1, "null", "3");
-        driver.flushState();
-        driver.process(topic1, "B", "4");
-        driver.flushState();
-        driver.process(topic1, "NULL", "5");
-        driver.flushState();
-        driver.process(topic1, "B", "7");
-        driver.flushState();
-
-        assertEquals(
-            asList(
-                "1:0+1 (ts: 0)",
-                "1:0+1-1 (ts: 0)",
-                "1:0+1-1+1 (ts: 0)",
-                "2:0+2 (ts: 0)",
-                  //noop
-                "2:0+2-2 (ts: 0)", "4:0+4 (ts: 0)",
-                  //noop
-                "4:0+4-4 (ts: 0)", "7:0+7 (ts: 0)"),
-            supplier.theCapturedProcessor().processed);
+        try (
+            final TopologyTestDriver driver = new TopologyTestDriver(
+                builder.build(),
+                mkProperties(mkMap(
+                    mkEntry(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy"),
+                    mkEntry(StreamsConfig.APPLICATION_ID_CONFIG, "test"),
+                    mkEntry(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory("kafka-test").getAbsolutePath())
+                )),
+                0L)) {
+            final ConsumerRecordFactory<String, String> recordFactory =
+                new ConsumerRecordFactory<>(new StringSerializer(), new StringSerializer(), 0L, 0L);
+
+            driver.pipeInput(recordFactory.create(topic1, "A", "1"));
+            driver.pipeInput(recordFactory.create(topic1, "A", (String) null));
+            driver.pipeInput(recordFactory.create(topic1, "A", "1"));
+
+            driver.pipeInput(recordFactory.create(topic1, "B", "2"));
+
+            driver.pipeInput(recordFactory.create(topic1, "null", "3"));
+
+            driver.pipeInput(recordFactory.create(topic1, "B", "4"));
+
+            driver.pipeInput(recordFactory.create(topic1, "NULL", "5"));
+
+            driver.pipeInput(recordFactory.create(topic1, "B", "7"));
```

Comment:
nit: why do many blank lines??

Reply:
ah, it was `s/driver.flushState();//g` :) I'll remove the blank lines.